ajax/libs/lockr/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "lockr",
+  "filename": "lockr.min.js",
+  "version": "0.8.4",
+  "description": "Lockr is an extremely lightweight library (<2kb when minified), designed to facilitate how you interact with localStorage.",
+  "author": {
+    "name": "Dimitris Tsironis",
+    "email": "tsironis.dimitris@gmail.com"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/tsironis/lockr.git"
+  },
+  "license": "MIT",
+  "homepage": "https://github.com/tsironis/lockr#readme",
```

Comment:
@x09326 you should try `pnpm` which will not touch the `package.json` from upstream.
See https://github.com/rstacruz/pnpm

cc @cdnjs/intern2